client/node.go

```diff
@@ -205,6 +212,12 @@ func (n *NodeClient) Pools(ctx context.Context) (pools []pkg.PoolMetrics, err er
 	return
 }
 
+func (n *NodeClient) GPUs(ctx context.Context) (gpus []GPU, err error) {
```

Comment:
If the node can have multiple GPUs, the original plan was the user will be reserving the full node + the gpu (i assume it was just one) or can it be multiple ?

Reply:
there can be many GPUs available on one node. To have access to the GPUs you will need to rent the nodes. But once it's rented you can choose which GPU(s) go to which VMs. so you can actually run multiple VMs on the same node each with a sub-set of the GPUs available on th node